acceptance/setup/pre_suite/80_add_dev_repo.rb

```diff
@@ -1,4 +1,4 @@
-install_opts = options.merge( { :dev_builds_repos => ["PC1"] })
+install_opts = options.merge( { :dev_builds_repos => ["puppet"] })
```

Comment:
The only thing is I don't know how beaker will handle this change, if it will do the right thing or now